docs/backends.md

```diff
@@ -15,7 +15,7 @@ in most of its commands.
 Each backend has its own parameters that are required to use it. Those
 parameters can be set as command line options or environment variables; the
 later is the **recommended solution** for sensible data such as service
-credentials. If we consider the `os_username` (OpenStack user name) parameter
+credentials. If we consider the `os_username` (OpenStack username) parameter
```

Comment:
```suggestion
credentials. If we consider the `os_username` (OpenStack user name) parameter
```

"user name" seems to be the way they present it here:

https://docs.openstack.org/python-openstackclient/pike/cli/command-objects/user.html

Reply:
The variable is, in fact, named `os_username`, not `os_user-name`.
I fixed up your comment.